bumble/avdtp.py

```diff
@@ -551,7 +586,9 @@ def __str__(self):
 
 # -----------------------------------------------------------------------------
 class MediaCodecCapabilities(ServiceCapabilities):
-    def init_from_bytes(self):
+    media_codec_information: Union[bytes, SupportsBytes]
+
```

Comment:
missing `media_type` and `media_codec_type`?

Reply:
They can be inferred from `__init__`, but I added hints still.